controllers/cloud/internal/cloud_tool/interface.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2023 yxxchange@163.com.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cloud_tool
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	ntf "github.com/labring/sealos/controllers/common/notification/api/v1"
+	"github.com/labring/sealos/pkg/utils/logger"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	cl "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Cloud interface {
+	setCloudArgs(method string, url string)
```

Comment:
The lowercase initials of interface methods cannot be referenced by other packages.